examples/waiting-for-goroutines-to-finish/waiting-for-goroutines-to-finish.go

```diff
@@ -0,0 +1,47 @@
+// To wait for multiple goroutines to finish, we can
+// use a sync.WaitGroup.
+
+package main
+
+import (
+    "fmt"
+    "math/rand"
+    "sync"
+    "time"
+)
+
+// This is the function we'll run in every goroutine.
+// wg is the WaitGroup it uses to notify that it's done.
```

Comment:
variables form the code should be monotyped: "`wg` is the WaitGroup" etc.

Reply:
Removed this line since it's too obvious, IMHO and mentioned later too.